static/app.js

```diff
@@ -31,17 +36,124 @@ $(function() {
         )
     }
 
+    /** helper function to process brewery API response data
+     * takes response data and dataType as parameters, turns response data into an results object to be rendered
+     * dataType is in regards to autocomplete or normal results
+     */
 
-    if (!sessionStorage.getItem('latitude') && !sessionStorage.getItem('longitude') && 'geolocation' in navigator) {
-        getLocation()
+    function processBreweries(data, dataType){
+        const breweriesResults = Object.values(data).map((brewery) => {
+            return createBrewery(brewery, dataType)
```

Comment:
nit: you don't need the return here. you can just `.map(brewery => createBrewery...`

Reply:
i noticed that this doesn't work without the return statement so leaving there for now.